cmd/test/main.go

```diff
@@ -222,9 +222,9 @@ func New(ctx *TestCommandContext) *cobra.Command {
 // AddFlags registers flags for a cli
 func (flags *TestCommandFlags) AddFlags(cmd *cobra.Command) {
 	if ciContext.Extract() != nil && ciContext.Extract().CIMetadata.ShouldHideEmojis {
-		cmd.Flags().StringVarP(&flags.Output, "output", "o", "simple", "Define output format")
+		cmd.Flags().StringVarP(&flags.Output, "output", "o", "simple", "Define output format (json, yaml, xml, JUnit)")
```

Comment:
some of the output options are missing (the issue description was wrong).
You should use similar code to the one in this line:
https://github.com/datreeio/datree/blob/4e0854a6cef47ef922713e5678d76c0e2c49a75a/cmd/test/main.go#L78

`strings.Join(evaluation.ExplicitOptionOptions, ", ")`

consider extracting it into a single function in `/bl/evaluation/output_flag.go`

Reply:
I have addressed the comment. It is good for an another round of review.